pvlib/modelchain.py

```diff
@@ -413,11 +413,7 @@ def __setattr__(self, key, value):
         super().__setattr__(key, value)
 
     def __repr__(self):
-        # once per MC
-        mc_attrs = ['weather', 'solar_position', 'airmass', 'tracking', 'aoi',
-                    'aoi_modifier', 'total_irrad', 'spectral_modifier',
-                    'effective_irradiance', 'cell_temperature', 'diode_params',
-                    'dc', 'dc_ohmic_losses', 'losses', 'ac']
+        mc_attrs = dir(self)
```

Comment:
Using `dir` sorts the list of attributes alphabetically instead of conceptually.  Not necessarily a downside.

Reply:
Yeah, once I saw the alphabetized list my thought was "that's easier to use"